### PR TITLE
Rewrite highlighting

### LINKED ISSUE #34: Fix javascript syntax highlighting
~~When rendering the markdown files into html, the js styling doesn't work. What do you think about only using the markdown files? We can remove all the build code and just focus on writing the markdown files that already looks nice on github with respect to js style etc.~~

Fix the syntax highlighting in the built html

### PR DESCRIPTION
**Pull request by ralphtheninja**
Closes #34 

Before:

Screenshot from 2022-02-06 18-21-33
Screenshot from 2022-02-06 18-21-58

After:

Screenshot from 2022-02-06 18-20-38
Screenshot from 2022-02-06 18-21-06

### CODE REVIEW

**Comment by hackergrrl** [comment]
Sorry, no capacity to review PRs. 👎🏻

File: problems/02.md [hunk 1]
```diff
@@ -12,20 +12,20 @@ It's a good idea to keep [the hypercore README](https://github.com/mafintosh/hyp
...
+```sh
+$ git clone https://gist.github.com/okdistribute/6d2d11e43977054d0613f4714cc48076 kappa-workshop
 ```
 
 Then, use your terminal to change the current working directory to the `kappa-workshop`:
 
-```
-cd kappa-workshop
+```sh
+$ cd kappa-workshop
```
Comment (cblgh) [inline problems/02.md]: having the prefix `$` makes it harder to copy-paste no? at least if it's not added as a css attribute e.g. `code pre:before { text: '$';}` or similar

**Comment by cblgh** [comment]
lgtm i guess :) i'm assuming the pages in `build/` that were deleted will be added as part of a deployment or in a separate commit; otherwise i fear the github.io page might not work as expected

File: problems/02.md (hunk 1 above)
Comment (ralphtheninja) [reply problems/02.md]: Ah good catch!

**Comment by ralphtheninja** [comment]
> lgtm i guess :) i'm assuming the pages in `build/` that were deleted will be added as part of a deployment or in a separate commit; otherwise i fear the github.io page might not work as expected

They weren't deleted, it just looks like it on github, it was just a lot of lines removed. If you expand the diffs you'll see some green in there as well.

**Review by cblgh — APPROVED** [review]

**Pull request merged**